src/weapon/animations.h

```diff
@@ -0,0 +1,321 @@
+#include "game.h"
+
+#if defined(W_003) || defined(W_007)
```

Comment:
I think all the ifdefs in this file makes it hard to tell what belongs to which weapon. For example the `g_Hitboxes` part is actually nested inside these `#ifs`. I think Xeeynamo needs to comment about the assets tool first, but what if we handled this like the stage overlays and put the individual duplicated parts into .h files and #include those? I think that would be easier than a huge animations.h with a lot of #ifdefs

Reply:
Sure, I can pull them out into separate includes. Do you have a preferred naming convention? Maybe something like what @hohle did with `src/weapon/w_053_056.h`.

Reply:
I think `anim_003_007.h` would be fine for example